doc/dev/installation.rst

```diff
@@ -116,6 +119,9 @@ The essential steps are:
         cd catalyst
         pip install -r requirements.txt 
 
+        # (Only for conda users) Install the latest C++ library
+        conda install -c conda-forge libstdcxx-ng
+
```

Comment:
We should validate that this works on mac as well, so far it's only been tested on linux.

Reply:
I got this:

```
❯ conda install -c conda-forge libstdcxx-ng
Collecting package metadata (current_repodata.json): done
Solving environment: failed with initial frozen solve. Retrying with flexible solve.
Collecting package metadata (repodata.json): done
Solving environment: failed with initial frozen solve. Retrying with flexible solve.

PackagesNotFoundError: The following packages are not available from current channels:

  - libstdcxx-ng

Current channels:

  - https://conda.anaconda.org/conda-forge/osx-arm64
  - https://conda.anaconda.org/conda-forge/noarch
  - https://repo.anaconda.com/pkgs/main/osx-arm64
  - https://repo.anaconda.com/pkgs/main/noarch
  - https://repo.anaconda.com/pkgs/r/osx-arm64
  - https://repo.anaconda.com/pkgs/r/noarch
  - https://conda.anaconda.org/anaconda/osx-arm64
  - https://conda.anaconda.org/anaconda/noarch

To search for alternate channels that may provide the conda package you're
looking for, navigate to

    https://anaconda.org

and use the search bar at the top of the page.
```

Reply:
Good to know, so it should feature in the linux tab only then :)

Reply:
Is there a substitute command that needs to go here?

Reply:
It looks like libstdcxx-ng is linux only. I do not find any counterpart of that for mac.